src/main/config/secor.common.properties

```diff
@@ -381,3 +381,20 @@ secor.file.age.youngest=true
 # Class that manages metric collection.
 # Sending metrics to Ostrich is the default implementation.
 secor.monitoring.metrics.collector.class=com.pinterest.secor.monitoring.OstrichMetricCollector
+
+# Row group size in bytes for Parquet writers. Specifies how much data will be buffered in memory before flushing a
+# block to disk. Larger values allow for larger column chinks which makes it possible to do larger sequential IO.
+# Should be aligned with HDFS blocks. Defaults to 128MB in Parquet 1.9.
+parquet.block.size=134217728
+
+# Page group size in bytes for Parquet writers. Indivisible unit for columnar data. Smaller data pages allow for more
+# fine grained reading but have higher space overhead. Defaults to 1MB in Parquet 1.9.
+parquet.page.size=1048576
+
+# Enable or disable dictionary encoding for Parquet writers. The dictionary encoding builds a dictionary of values
+# encountered in a given column. Defaults to true in Parquet 1.9.
+parquet.enable.dictionary=true
+
+# Enable or disable validation for Parquet writers. Validates records written against the schema. Defaults to false in
+# Parquet 1.9.
+parquet.validation=false
```

Comment:
it's not a new character, just a notification that there is no newline. I added a new line and squashed